### PR TITLE
Added Backend resources

### PR DESCRIPTION
**Pull request by Sophyia7**
I included resources for backend languages and their frameworks.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,7 +54,9 @@ To add resources or to contribute read the [Contribution-guide](https://github.c
   * [CSS](#css)
   * [JavaScript](#javascript)
   * [React](#react)
-  * [Practice](#practice)  
+  * [Practice](#practice)
+- [Backend](#backend)
```
Comment (Ashish-khanagwal) [inline README.md:58]: Please, add `Backend` in alphabetical order.
Add it under the `Android Development` section

File: README.md [hunk 2]
```diff
@@ -161,6 +164,45 @@ To add resources or to contribute read the [Contribution-guide](https://github.c
   - :bulb: [Frontend Practice](https://www.frontendpractice.com/)
   - :bulb: [Codewell](https://www.codewell.cc/)
   - :bulb: [Javascript30](https://javascript30.com/)
+
+---
+
+## Backend
```
Comment (Ashish-khanagwal) [inline README.md:170]: add these resources in alphabetical order.
Add `Backend` section under `Android Development` section

**Review by Ashish-khanagwal — CHANGES_REQUESTED** [review]

**Comment by Sophyia7** [comment]
Alright.

**Comment by Ashish-khanagwal** [comment]
Inform me, once you resolve the changes

**Comment by Sophyia7** [comment]
Should I also add the resources under backend alphabetically? 

Like NodeJs before Python?

**Comment by Ashish-khanagwal** [comment]
> Should I also add the resources under backend alphabetically?
> 
> Like NodeJs before Python?

No, It's not necessary.

**Review by Ashish-khanagwal — APPROVED** [review]
Looks good to me

Thank you for contributing

**Pull request merged**